### DISCUSSION TITLE
Docker Image - missing entrypoint.sh

### DISCUSSION THREAD

**Discussion by ghost** [lllyasviel/Fooocus#2836] [Q&A] [answered]
Hi I built the docker image following the instructions but although the contains are created they fail to find /content/entrypoint.sh.  Any ideas what is causing this?  I am quite new to docker so relied on following the three steps you give in the doc.

**Comment by mashb1t** [comment]
Hey @Tre4Bax, just to nake sure: you're using the official docs (https://github.com/lllyasviel/Fooocus/blob/main/docker.md) and have not modified the Dockerfile, right?

**Comment by ghost** [comment]
Yes.   I just followed through what was in the Docker.md document from the main Github page.  it was so simple I was surprised when it did not work,  which is why I suspect my Docker Desktop setup somehow.  This is the second thing I've tried to add to docker so am a real newbie at it, though I have worked in IT for far too long so not a newbie at computers in general.

**Reply by mashb1t** [reply]
@Tre4Bax fixed in PR https://github.com/lllyasviel/Fooocus/pull/2843, please pull and build again

```
git pull
docker compose build
docker compose up
```

**Comment by ghost** [comment]
Sadly something still isn't working for me.  I just tried clearing the folder completely and re-cloning too just in case something had been left behind.

![image](https://github.com/lllyasviel/Fooocus/assets/41943699/3d00e0c5-cd6f-45d4-b41e-21084e6694b3)

**Reply by mashb1t** [reply, marked as answer]
Make sure the line breaks are in LF and not CLRF format (might have been reset when you saved the file). You can change this in your editor of choice, e.g. Sublime, Atom or PyCharm.

**Comment by ghost** [comment]
hi mashbit,  I've just done that but still no change.  I am using VSCode and made sure I changed to LF and then saved and I noted it added that as a change in Git so pretty sure that is okay.   I did this with the dockerfile,   were there other files I should check this on?

**Reply by mashb1t** [reply]
The Dockerfile doesn't need adjustments, just the entrypoint.sh
Please make sure it had LF line breaks.

**Comment by ghost** [comment]
Thanks mashbit,  that's it sorted.  The entrypoint.sh was crlf changed to lf and it is now progressing to doing some downloading etc.  will leave it and let it get on but it is looking good.
